<launch>
    <node name="line_extractor_node" pkg="sb_pointcloud_processing" type="line_extractor_node" output="screen">
        <!-- degree of polynomial for line of best fit -->
        <param name="degree_polynomial" value="2" type="int" />
        <!-- regularization constant for line of best fit -->
        <param name="lambda" value="0" type="double" />
        <!-- density parameters for DBSCAN -->
        <param name="min_neighbours" value="13" type="int" />
        <param name="radius" value="0.1" type="double" />

        <!-- rviz parameters -->
        <!-- frame id should match the one of "/height_filter/output" -->
        <param name="frame_id" value="base_link" />
        <!-- scale of the points displayed in rviz -->
        <param name="scale" value="0.01" />

        <!-- subscribe to /height_filter/output for input point cloud -->
        <remap from="/input_pointcloud" to="/voxel_grid/output" />
    </node>
</launch>
